generatedocs/data/config/scheduler/examples.md

```diff
@@ -1,5 +1,21 @@
+### Using `reconnectsettings`
+
+If `reconnectsettings.reconnect` is enabled, the following is attempted:
+
+* Re-connect the WebSocket.
```

Comment:
Are the actions carried out in the specified order - should this perhaps be a numbered list?

Reply:
ehm.. yes actually they are, was just listing thinks that would happen, but yes they're even in order so numbered list makes sense.

Reply:
or now the bullets are re-ordered to correct order in which they happen. Also added a small addition as logic just changed slightly.